eva/catalog/catalog_manager.py

```diff
@@ -388,50 +429,50 @@ def check_table_exists(self, database_name: str, table_name: str):
     def get_all_udf_entries(self):
         return self._udf_service.get_all_udfs()
 
-    def get_video_metadata_table(
+    def get_media_metainfo_table(
```

Comment:
Maybe add a comment on the difference between `metainfo` and `metadata`.

Reply:
The catalog's current naming convention is ineffective, and I am working on a new pull request to improve it.